packages/smooth_app/lib/l10n/app_en.arb

```diff
@@ -1,346 +1,386 @@
-    },
-    "product_improvements_count": "{count,plural, =1{One possible improvement} other{{count} possible improvements}}",
-    "@product_improvements_count": {
-        "description": "x possible improvements",
-        "placeholders": {
-            "count": {}
-        }
-    },
-    "products_pasted": "products pasted",
-    "@products_pasted": {},
-    "no_prodcut_in_list": "There is no product in this list",
-    "@no_product_in_list": {},
-    "error_occurred": "An error occurred",
-    "@error_occurred": {},
-    "keyword_search": "(keyword search)",
-    "category_search": "(category search)",
-    "my_list": "(my list)",
-    "scan": "Scan",
-    "recently_seen_products": "My recently seen products",
-    "unknown_product_list": "Unknown product list:",
-    "not_found": "not found:",
-    "no_product": "no product",
-    "one_product": "1 product",
-    "x_products": "products",
-    "scan_contribute": "CONTRIBUTE",
-    "@scan_contribute": {
-        "description": "Switch on top of the Scanner to choose between to scan or add products"
-    },
-    "incompatible": "Incompatible",
-    "pct_match": "{pct}% match",
-    "@pct_match": {
-        "placeholders": {
-            "pct": {}
-        }
-    },
-    "scan_choose": "CHOOSE",
-    "@scan_choose": {
-        "description": "Switch on top of the Scanner to choose between to scan or add products"
-    },
-    "reloaded_with_new_preferences": "Reloaded with new preferences",
-    "no_product_in_section": "There is no product in this section",
-    "what_are_you_looking_for": "What are you looking for?",
-    "chached_results_from": "Cached results from:",
-    "plural_ago_minutes": "{count,plural, =0{less than a minute ago} =1{one minute ago} other{{count} minutes ago}}",
-    "@plural_ago_minutes": {
-        "description": "Cached results from: x minutes ago",
-        "placeholders": {
-            "count": {}
-        }
-    },
-    "plural_ago_hours": "{count,plural, =1{one hour ago} other{{count} hours ago}}",
-    "@plural_ago_hours": {
-        "description": "Cached results from: x hours ago",
-        "placeholders": {
-            "count": {}
-        }
-    },
-    "plural_ago_days": "{count,plural, =1{one day ago} other{{count} days ago}}",
-    "@plural_ago_days": {
-        "description": "Cached results from: x days ago",
-        "placeholders": {
-            "count": {}
-        }
-    },
-    "plural_ago_weeks": "{count,plural, =1{one week ago} other{{count} weeks ago}}",
-    "@plural_ago_weeks": {
-        "description": "Cached results from: x weeks ago",
-        "placeholders": {
-            "count": {}
-        }
-    },
-    "plural_ago_months": "{count,plural, =1{one month ago} other{{count} months ago}}",
-    "@plural_ago_months": {
-        "description": "Cached results from: x months ago",
-        "placeholders": {
-            "count": {}
-        }
+
+  "@Utils": {},
```

Comment:
These are just like comments for deviding the Strings, (there is no other way to add comments)